fingerprint2.js

```diff
@@ -111,7 +111,7 @@
         var values = [];
         that.each(newKeys, function(pair) {
           var value = pair.value;
-          if (typeof pair.value.join !== "undefined") {
+          if (pair.value && typeof pair.value.join !== "undefined") {
```

Comment:
Please let me know in what cases pair can be null? That does not fix the root cause of the error but only hides it